Corona-Warn-App/src/main/res/values-de/risk_details_strings.xml

```diff
@@ -19,13 +19,13 @@
     <!-- XTXT: risk details - health department info -->
     <string name="risk_details_behavior_body_health_department">Bei Fragen zu Quarantäne-Maßnahmen wenden Sie sich bitte an Ihr Gesundheitsamt.</string>
     <!-- XTXT: risk details - health department info link label -->
-    <string name="risk_details_behavior_body_health_department_label">Fragen zu Quarantäne-Maßnahmen</string>
+    <string name="risk_details_behavior_body_health_department_label">Mehr Informationen finden Sie in den FAQ zu Quarantäne-Maßnahmen.</string>
```

Comment:
@jurajkusnier FYI

Reply:
Why will it break the link? Anything I need to do on my side?

Reply:
Yes, you can see the line above the changed line with the key `risk_details_behavior_body_health_department` 
```xml
 <string name="risk_details_behavior_body_health_department">Bei Fragen zu Quarantäne-Maßnahmen wenden Sie sich bitte an Ihr Gesundheitsamt.</string>
 ```

It has the old term `Fragen zu Quarantäne-Maßnahmen` in it , and it will not match the new change therefore the link wont't work 

to fix it, you have to update` risk_details_behavior_body_health_department` also.

Reply:
Ok, I see. Sorry for messing things up.
risk_details_behavior_body_health_department should no longer have a hyperlink, so no label require for it.
Instead, we need an additional string "Mehr Informationen finden Sie in den FAQ zu Quarantäne-Maßnahmen." and this one then needs a label because it should be a hyperlink.
How should we proceed here? Should I revert the change and ask a developer to do it?

Reply:
Ok, I understand, sorry for messing things up.
Then it is wrong anyhow. 
risk_details_behavior_body_health_department should not have a hyperlink, it's just a simple text. (so no label here)
We need an additional string "Mehr Informationen finden Sie in den FAQ zu Quarantäne-Maßnahmen." And for this we need a label with the same text.
How are we going to proceed?

Reply:
I've moved "Mehr Informationen finden Sie in den FAQ zu Quarantäne-Maßnahmen." now into risk_details_behavior_body_health_department, so this should work now.

Reply:
Thank you!, works fine